frontend/model/contracts/group.js

```diff
@@ -1810,16 +1746,26 @@ sbp('chelonia/defineContract', {
           encryptionKeyId,
           ...actorID === memberID && {
             hooks: {
+              postpublish: () => {
+                sbp('chelonia/contract/retain', chatRoomId)
+              },
               onprocessed: () => {
                 sbp('state/vuex/commit', 'setCurrentChatRoomId', { groupId: contractID, chatRoomId })
               }
             }
           }
         }).catch(e => {
+          if (e.name === 'GIErrorUIRuntimeError' && e.cause?.message?.endsWith('is already part of')) {
```

Comment:
Generally speaking, `GIErrorUIRuntimeError`s are intended to be displayed to the user, i.e. their cause is a user-facing string.

At the moment, this string isn't being translated (wrapped in an `L` function). But it probably should be. And if/when that happens, you won't be able to do this comparison, because you'll be checking against a translation.

So it's probably better/safer to check some sort of other parameter that is set on the error, like a `.type` or something, that is set to some internal non-userfacing value.

Reply:
> At the moment, this string isn't being translated (wrapped in an L function). But it probably should be. And if/when that happens, you won't be able to do this comparison, because you'll be checking against a translation.

I'm not checking the `message` of the `GIErrorUIRuntimeError`, but rather the `message` of the `cause` of the `GIErrorUIRuntimeError`, which isn't wrapped in an `L` function.

Reply:
Using the `name` attribute now